buzzmobile/constants.yaml

```diff
@@ -3,16 +3,27 @@
 # These all relate to the shape of the interal representation of the world,
 # which is essentially a numpy array (or image) of `image_width` by 
 # `image_height` with a scale factor of `pixels_per_m`, corresponding to the
-# number of pixels per each real-world meter
+# number of pixels per each real-world meter. We also define `road_width`, a
+# measurement of the width of the road in meters. This will be scaled to an
+# appropriate size using `pixels_per_m` during construction
+# of the GPS model image.
 image_width: 600
 image_height: 800
 pixels_per_m: 80
+road_width: 0.5
```

Comment:
Put this and its comment in a new block. It's not related to image things at all.